sys-libs/libcxx/libcxx-11.0.0.ebuild

```diff
@@ -100,6 +106,8 @@ multilib_src_configure() {
 			fi
 		fi
 	fi
+	# Darwin has libunwind in its system
+	[[ ${CHOST} == *-darwin* ]] && want_gcc_s=OFF
```

Comment:
I am somewhat doubtful about overriding only one of the three variables that are set above. I suppose the above logic should yield reproducible results on Darwin, correct?

Reply:
Yes. I am concerned about that as well. I'm reintroducing some things from older prefix ebuilds.
This might not be needed. I'll revisit this to see.

Reply:
I reworked this so that it is hopefully clearer.